xwiki-platform-core/xwiki-platform-websocket/src/test/java/org/xwiki/websocket/script/WebSocketScriptServiceTest.java

```diff
@@ -109,4 +109,11 @@ void urlWithException() throws Exception
             "Failed to create WebSocket URL for [test]. Root cause is [MalformedURLException: Invalid server URL!].",
             this.logCapture.getMessage(0));
     }
+
+    @Test
+    void urlWithEmptyWebAppPath() {
```

Comment:
Rather than "empty", I think it's more correct to mention  "Root" servlet context.